src/frontend/src/pages/NumberTracing.tsx

```
@@ -213,6 +213,70 @@ const NumberTracingGame = memo(function NumberTracingGameComponent({
     onFrame: handleHandTrackingFrame,
   });
 
+  // Hand tracking frame handler
+  const handleHandTrackingFrame = useCallback(
```

Biome (2.4.6)

[error] Shouldn't redeclare 'handleHandTrackingFrame'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)